common/src/main/java/com/microsoft/identity/common/exception/ErrorStrings.java

```diff
@@ -335,4 +335,8 @@ private ErrorStrings() {
      */
     public static final String UNSUPPORTED_BROKER_VERSION = "unsupported_broker_version";
 
+    /**
+     * Decryption failed.
+     */
+    public static final String DECRYPTION_FAILED = "decryption_failed";
```

Comment:
>decryption_failed [](start = 52, length = 17)

Is this going to be consumed by client apps?

Reply:
It should be handled by broker.